.metwork-framework/cheatsheet.md

```diff
@@ -92,7 +92,7 @@ As `mfserv` unix user:
 | --- | --- |
 | **`config.ini`** | main plugin configuration file |
 | **`Makefile`** | build configuration file (you probably don't need to touch this unless you have specific build directives to add to the `custom::` target) |
-| `crontab` | plugin crontab (use something like `* * * * * {{MFSERV_HOME}}/bin/cronwrap.sh -l -e -- plugin_wrapper {YOUR_PLUGIN_NAME} {YOUR_COMMAND} >>{{MODULE_RUNTIME_HOME}}/log/your_command.log 2>&1` for each line to execute your command in your plugin environement) |
+| `crontab` | plugin crontab (use something like `* * * * * {{MFSERV_HOME}}/bin/cronwrap.sh -l -e -- plugin_wrapper {YOUR_PLUGIN_NAME} {YOUR_COMMAND} >>your_command.log 2>&1` for each line to execute your command in your plugin environement) |
```

Comment:
this doc has to be fixed as it does not use --log-capture-to

Reply:
Ok. But this file is under resources. We won't fix it in mfserv.